corehq/apps/locations/templates/locations/manage/locations.html

```diff
@@ -57,17 +57,15 @@
       <div class="col-sm-12">
         <div class="btn-toolbar">
 
-          {% if request.can_access_all_locations %}
-            <a class="btn btn-info"
-              {% if not has_location_types %}
-               disabled
-              {% else %}
-               href="{{ bulk_download_url }}"
-              {% endif %}
-            >
-              <i class="fa fa-cloud-download"></i> {% trans 'Download Organization Structure' %}
-            </a>
-          {% endif %}
+          <a class="btn btn-info"
+            {% if not has_location_types %}
+             disabled
+            {% else %}
+             href="{{ bulk_download_url }}"
+            {% endif %}
+          >
+            <i class="fa fa-cloud-download"></i> {% trans 'Download Organization Structure' %}
```

Comment:
Have we tested whether a location-restricted user can't do this?

Reply:
@kaapstorm The location restricted user _can_ download the Organization Structure, but only from the location the user is assigned to and down (child locations).